perfkitbenchmarker/beam_benchmark_helper.py

```diff
@@ -275,10 +290,13 @@ def _BuildPythonCommand(benchmark_spec, modulename, job_arguments):
   return cmd
 
 
+def _GetGradleCommand():
+  return FLAGS.gradle_binary if FLAGS.gradle_binary else _GetBeamDir() + '/gradlew'
```

Comment:
return FLAGS.gradle_binary or _GetBeamDir() + '/gradlew'

Reply:
you may also use `os.path.join` to build `gradlew` path

Reply:
ok